meson.build

```diff
@@ -564,13 +573,3 @@ subdir('docs')
 if build_daemon and libsystemd.found()
   meson.add_install_script('meson_post_install.sh', systemdunitdir)
 endif
-
-makensis = find_program('makensis', required : host_machine.system() == 'windows')
```

Comment:
I think you want to add a `find_program('wixl', required: host_machine.system == 'windows')` here still

Reply:
I figured it was just a non-installed configure.in-style replacement -- your logic is just to avoid doing that on win32 if it's not going to be used?

Reply:
My thought is that if the build script needs it, then it should be something that we check for in meson.build so we can message accordingly.

Reply:
Well, the CI script needs it, but not generally for building win32...